mobile-save-for-later/src/main/scala/com/gu/sfl/local/LocalServer.scala

```diff
@@ -0,0 +1,35 @@
+package com.gu.sfl.local
+
+import cats.data.Kleisli
+import cats.effect.{ExitCode, IO, IOApp}
+import com.gu.sfl.lambda.{FetchArticlesLambda, LambdaRequest, LambdaResponse, SaveArticlesLambda}
+import org.http4s.blaze.server.BlazeServerBuilder
+import org.http4s.dsl.io._
+import org.http4s.implicits.http4sKleisliResponseSyntaxOptionT
+import org.http4s.server.Router
+import org.http4s.{HttpRoutes, Request, Response}
+
+object LocalServer extends IOApp {
```

Comment:
You could consider moving this `LocalServer` class from `src/main/scala` to `src/test/scala` - it could still be run as a CLI class (an example of this is [`SectionGenerator` in Ophan](https://github.com/guardian/ophan/blob/72ab516ecf12de6f0c99f495c9eef41a50f2bfaf/shared-lib/src/test/scala/SectionGenerator.scala#L21)), but that way it & it's dependencies (eg `http4s-server`) could be kept in the `test` scope, so that the assembled jar that's used for the AWS Lambda runtime is smaller and does not contain those unnecessary deps - makes Lambda startup faster and cheaper!

However, this may all be slightly moot at the moment, as it turns out that `identity-auth-core` pulls in `http4s-server` anyway (since https://github.com/guardian/identity/pull/1571), so currently there wouldn't be substantial benefit to carefully removing `LocalServer`'s dependencies from the runtime - they'd get added anyway by `identity-auth-core`!